.github/ISSUE_TEMPLATE/summit-topic-proposal.md

```diff
@@ -0,0 +1,24 @@
+---
+name: "\U+270F Summit Topic Proposal"
+about: Submit a topic proposal for a session at the Collab Summit
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+Thank you! You are submitting a topic for the next Collaborator's Summit, Berlin 2019! After you send this, feel free to compile https://github.com/nodejs/summit/issues/149 with some information. That form will help us in collecting the information for the agenda.
```

Comment:
Would be better if we keep this template generic for all Collab Summits so that we don't need to update it every time.
Skip referring "Berlin, 2019"
Skip link to "issues/149"

Maybe we can make one google form for all collab summits. and add an option to select the event (like a dropdown selection)

Reply:
Thank you for noticing that it was event-specific!

The template instructions being specific is meant to reassure folks that this template is current and meant to be used :) There's already some confusion about whether to submit through the Form or through Github Issues.

With the changes to the Foundation, we're very likely to need updates each time the Collab Summit is run. I think updating this template will be trivial when necessary.